server/src/services/domain/localization.service.ts

```diff
@@ -42,20 +43,21 @@ export function getLocalizedMessages(
         | Array<Array<string> | string>
         | Array<unknown>
 ): Array<string> | string {
+    const loc = locale ?? DEFAULT_LOCALE;
```

Comment:
you do not need to do that, i18n will do it by itself.  It will take default locale defined in `src\index.ts`

Reply:
removed